minimal/do

```diff
@@ -103,10 +103,10 @@ if [ -z "$DO_BUILT" -a "$_cmd" != "redo-whichdo" ]; then
 	for d in redo redo-ifchange redo-whichdo; do
 		ln -s "$REDO" "$DO_PATH/$d"
 	done
-	[ -e /bin/true ] && TRUE=/bin/true || TRUE=/usr/bin/true
 	for d in redo-ifcreate redo-stamp redo-always redo-ood \
 	    redo-targets redo-sources; do
-		ln -s $TRUE "$DO_PATH/$d"
+	        echo : > "$DO_PATH/$d"
+		chmod a+rx "$DO_PATH/$d"
```

Comment:
The created file should actually have a leading #!/ line.  Although executable files without #!/ seem to work when run from the shell, they don't work when run from elsewhere, which (probably rather perverse) .do scripts might do.

     $ echo : >foo
     $ chmod a+x foo
     $ ./foo
     $ python
    Python 2.7.13 (default, Sep 26 2018, 18:42:22) 
    >>> import os
    >>> os.system('./foo')
    0
    >>> os.execv('./foo', ['./foo'])
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    OSError: [Errno 8] Exec format error